pyodata/v2/service.py

```diff
@@ -1240,12 +1252,25 @@ def filter(self, *args, **kwargs):
         return self
 
 
-class ListWithTotalCount(list):
-    """A list with the additional property total_count"""
+class ListWithProgressIndication(list):
+    """
```

Comment:
Are you sure you need to rename the class?

Reply:
I did it because it is more descriptive now. Not sure if it outweighs the API breakage however.